src/core_plugins/input_control_vis/public/control/create_search_source.js

```diff
@@ -1,13 +1,15 @@
 
-export function createSearchSource(kbnApi, initialState, indexPattern, aggs, useTimeFilter) {
+export function createSearchSource(kbnApi, initialState, indexPattern, aggs, useTimeFilter, filters = []) {
   const searchSource = new kbnApi.SearchSource(initialState);
   // Do not not inherit from rootSearchSource to avoid picking up time and globals
   searchSource.inherits(false);
-  if (useTimeFilter) {
-    searchSource.filter(() => {
-      return kbnApi.timeFilter.get(indexPattern);
-    });
-  }
+  searchSource.filter(() => {
+    const activeFilters = [].concat(filters);
```

Comment:
I think we try to use ES2016 syntax for that kind of things: `const activeFilters = [...filters];`